### PR TITLE
Remove min-width from extended textarea

### PR DESCRIPTION
**Pull request by SaraBoutej**

### CODE REVIEW

File: platform-ui-skin/src/main/webapp/skin/less/core/components/Textarea/Style.less [hunk 1]
```diff
@@ -20,7 +20,6 @@
 
 .extended-textarea {
   textarea {
-    min-width: 100%;
```
Comment (boubaker) [inline platform-ui-skin/src/main/webapp/skin/less/core/components/Textarea/Style.less:23]: why deleting this? you can use css helpers to force switching this value to other value knowing that CSS helpers uses `!important`

Comment (SaraBoutej) [reply platform-ui-skin/src/main/webapp/skin/less/core/components/Textarea/Style.less:23]: ok

Comment (SaraBoutej) [reply platform-ui-skin/src/main/webapp/skin/less/core/components/Textarea/Style.less:23]: ok

Comment (SaraBoutej) [reply platform-ui-skin/src/main/webapp/skin/less/core/components/Textarea/Style.less:23]: ok

Comment (SaraBoutej) [reply platform-ui-skin/src/main/webapp/skin/less/core/components/Textarea/Style.less:23]: Not needed!! I'll delete the PR

**Pull request closed without merging**